### PR TITLE
Persist settings after updates

### LINKED ISSUE #441: After VS update all settings lost
After Visual Studio update. The DeepL API-Key and API-Url are lost.

### PR DESCRIPTION
**Pull request by jakehedlund**
Fixes #441. 

See issue for details. 

VS automatically changed the "GeneratedClassNamespace" attribute in `Settings.Settings` - I don't think this will cause any side effects so I left it as is.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/ResXResourceManager?pullRequest=475) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/ResXResourceManager?pullRequest=475) before we can accept your contribution.<br/><br/>:x: jakehedlund [sign now](https://cla.dotnetfoundation.org/dotnet/ResXResourceManager?pullRequest=475)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/ResXResourceManager?pullRequest=475) it.</sub>

File: src/ResXManager/MainViewModel.cs [hunk 1]
```diff
@@ -44,7 +44,18 @@ public MainViewModel(ResourceManager resourceManager, IConfiguration configurati
 
             try
             {
-                var folder = Settings.Default.StartupFolder;
+                var settings = Settings.Default;
```
Comment (tom-englert) [inline src/ResXManager/MainViewModel.cs:47]: That's for sure a lot of code duplication here...

**Comment by tom-englert** [comment]
How was this tested? Have you verified that this does not affect settings from other extensions?

**Comment by jakehedlund** [comment]
Good points. I did not test the modified extension, just the standalone app. However, it's my understanding that since the `Settings.Properties` classes are auto-generated and handled per-project, each project/class can only modify its own settings. So it should be safe (but yes, should be tested too).

File: src/ResXManager/MainViewModel.cs (hunk 1 above)
Comment (jakehedlund) [reply src/ResXManager/MainViewModel.cs:47]: Yeah it was quick and dirty. Fixed in latest.

**Comment by tom-englert** [comment]
Superseded by https://github.com/dotnet/ResXResourceManager/commit/5d8144b245d44fc430cd14249d28a6a1e2381a1d

**Pull request closed without merging**